docs/user/ppl/admin/prometheus_connector.rst

```diff
@@ -210,3 +209,71 @@ Example::
     | 9          | "2022-11-03 07:18:54"  | "/-/promql"                    | 400           | 192.15.2.1  | prometheus  |
     | 11         | "2022-11-03 07:18:64"  |"/-/metrics"                    | 500           | 192.15.2.1  | prometheus  |
     +------------+------------------------+--------------------------------+---------------+-------------+-------------+
+
+
+Prometheus Connector Table Functions
+==========================================
+
+`query_exemplars` Table Function
+----------------------------
+* This table function can be used to fetch exemplars of a query in a specific time range.
+* The function takes inputs similar to parameters mentioned for query exemplars api mentioned here: https://prometheus.io/docs/prometheus/latest/querying/api/
+* Arguments should be either passed by name or positionArguments should be either passed by name or position.
+    - `source=my_prometheus.query_exemplars('prometheus_http_requests_total', 1686694425, 1686700130)`
+    - `source=my_prometheus.query_exemplars(query='prometheus_http_requests_total', starttime=1686694425, endtime=1686700130)`
+Example::
+
+    > source=my_prometheus.query_exemplars('prometheus_http_requests_total', 1686694425, 1686700130)
+      "schema": [
```

Comment:
Maybe: split blocks?
It is possible to make it as an executable doctest?

Note: just detected that test for `query_range` isn't executed too.

Reply:
For query_range looking for a way to make starttime and endtime dynamic. We start prometheus instance on demand when we run doctests. The data available is only from the doctest start time.

Exemplars are not available with the prometheus instance that we spin up in doctests. 

It is hard to split the nested objects into blocks like other results. Is there any tool that you are aware of?